utilities/custom-clang-format.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+
+import sys
+import subprocess as proc
+
+# This script fixes an issue where clang-format puts the AMREX_ specifiers on the same line as the function name.
+# There is no option to not do this, so this script first turns those into comments, runs clang-format, and then
+# turns them back into specifiers.
+# A similar approach could be used to fix other clang-format limitations
+
+content = sys.stdin.read()
+content = content.replace("AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE\n", "/*AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE*/\n")
+content = content.replace("AMREX_GPU_DEVICE AMREX_FORCE_INLINE\n", "/*AMREX_GPU_DEVICE AMREX_FORCE_INLINE*/\n")
+proc_obj = proc.Popen(["clang-format", "--style=file"], stdin=proc.PIPE, stdout=proc.PIPE, stderr=proc.PIPE, text=True)
+proc_obj.stdin.write(content)
+content, error = proc_obj.communicate()
```

Comment:
Can you add an error handling here? Right now I am getting a blank output from this script (which I think is because of the `clang-format` invalid arguments issue), so it will be nice to know that there was an error in execution.